config/routes_into_teaching.yml

```diff
@@ -88,6 +97,15 @@ routes:
         answers: ["*"]
       - question: location
         answers: ["No"]
+    highlight:
```

Comment:
@spencerldixon I've tested this successfully re two highlighted routes on the same results page and seeing a highlighted route being displayed at the top when it wouldn't have been otherwise

Happy for you to remove this test data now!